fuse_examples/imaging/oai_example/downstream/segmentation_config.yaml

```diff
@@ -0,0 +1,46 @@
+defaults:
+  - _self_
+  - override hydra/hydra_logging: disabled
+  - override hydra/job_logging: disabled
+
+hydra:
+  output_subdir: null
+  run:
+    dir: .
+
+task : segmentation
+batch_size : 2
+accumulate_grad_batches : 1
+precision : 32
+backbone : unet3d
+num_classes : 7 # including background
+resize_to : [40, 224,224]
+csv_path : /dccstor/mm_hcls2/oai/code/data/seg_oai.csv #fill
+results_dir : /dccstor/mm_hcls2/oai/results #fill
+train_folds : [train]
+val_folds : [valid]
+test_folds : [test]
+split_pth : null #fill
+
+n_workers : 24
+clearml : false
+clearml_project_name : OAI/downstream
+
+experiment : debug_example
```

Comment:
I think a better name for the experiment is oai_segmentation. Also maybe change experiment name in the other configs as well.